time_offset_to_apply: 0.0 # millisecond
topics:
  - "/gt_box/stim320/imu"
  - "/gt_box/stim320/gyroscope_temperature"
  - "/gt_box/stim320/acceletometer_temperature"